schemas/provider-reference/provider-reference.json

```diff
@@ -20,36 +22,46 @@
               "enum": ["ein", "npi"]
             },
             "value": {
-              "type": "string"
+              "type": "string",
+              "minLength": 1
             }
           },
-          "required": [
-            "type",
-            "value"
-          ]
+          "required": ["type", "value"]
         }
       },
-      "required": [
-        "npi",
-        "tin"
-      ]
+      "required": ["npi", "tin"]
+    },
+    "provider_references": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "provider_group_id": {
+            "type": "number"
```

Comment:
I would like to see this change into a string.

anthem does something that makes an extremely large float

161.0025066629

for example and I worry that since it has no defined length we could run into Double parsing issues where it can't represent the number as a Double and hides rates.

161.0000000000025066629
161.0000000000025066630

getting parsed as 

161.0000000000025

Switching this to a string would keep the json parsing from becoming an issue.

Reply:
@pbrumm could this be solved by not allowing for float numbers by using an `integer` type instead? I'm trying to better understand why a value like "161.0000000000025066630" would be used in the first place.

Reply:
@mint-thompson Let's update this to be an `integer` for the time being.

Reply:
@shaselton-usds @mint-thompson  Agree that the integer definition would work better than the number